modules/core/src/main/scala/higherkindness/droste/package.scala

```diff
@@ -24,7 +41,16 @@ object `package` {
   type CVAlgebra[F[_], A]   = GAlgebra[F, Attr[F, A], A]
   type CVCoalgebra[F[_], A] = GCoalgebra[F, A, Coattr[F, A]]
 
+  /** A `Gather[F, S, A]` is a type alias for `(A, F[S]) => S`, that is a function
```

Comment:
Consider pulling some documentation from the source for gather/scatter: https://github.com/ekmett/recursion-schemes/pull/51. Also feel free to add some attribution!